src/crimson/osd/ops_executer.h

```diff
@@ -87,9 +94,38 @@ class OpsExecuter {
   seastar::chunked_fifo<std::unique_ptr<effect_t>> op_effects;
 
   template <class Context, class MainFunc, class EffectFunc>
-  auto with_effect(Context&& ctx, MainFunc&& main_func, EffectFunc&& effect_func);
+  auto with_effect_on_obc(
+    Context&& ctx,
+    MainFunc&& main_func,
+    EffectFunc&& effect_func);
 
   call_errorator::future<> do_op_call(class OSDOp& osd_op);
+  watch_errorator::future<> do_op_watch(
```

Comment:
Hmm, `errorator` is a bit lengthy. Consider switching to `ertr`. We might want to make a convention for the future.